src/interface.c

```diff
@@ -1650,7 +1702,7 @@ static void option_do_kill(struct nvtop_interface *inter) {
   if (inter->process.option_window.selected_row == 0)
```

Comment:
This is the cancel-specific bit.

Reply:
Oh, I got it. I thought that `"Cancel"` meant to kill process.